CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -809,7 +787,47 @@ public int DestroyDescriptorPoolResources() {
     
     // ========================> Pipelines <======================== \\
     
-    public int CreatePipeline() {
+    private int CreatePipelineLayout() {
+        CVKAssert(cvkDevice != null);
+        CVKAssert(cvkDevice.GetDevice() != null);
+               
+        int ret;       
+        try (MemoryStack stack = stackPush()) {      
+            // ===> PUSH CONSTANTS <===
+            VkPushConstantRange.Buffer pushConstantRange;
+            pushConstantRange = VkPushConstantRange.callocStack(1, stack);
+            pushConstantRange.stageFlags(VK_SHADER_STAGE_VERTEX_BIT);
+            pushConstantRange.size(VertexUniformBufferObject.SIZEOF);
+            pushConstantRange.offset(0);
+
+            // ===> PIPELINE LAYOUT CREATION <===                
+                
+            // Note: When using PushConstants you still need a DescriptorLayout 
```

Comment:
Looks like you removed the Descriptor Layouts and it still works? That's great, if so can you please remove this comment block?

Reply:
Done